### ISSUE TITLE
about gsplat aabb 

### ISSUE THREAD

**Issue by szhang963** [nerfstudio-project/nerfstudio#3319]
**Describe the bug**
I found the quality decreased when I did not use ‘auto_scale_poses’ to scale the poses for gsplat method. 
Though I adjust the scene box to adapt the poses, like this,
```
      poses_xyz = poses[:, :3, 3]
      aabb_scale = poses_xyz.abs().max(dim=0)[0] + 40
      aabb = torch.stack([-aabb_scale, aabb_scale], dim=0)
      scene_box = SceneBox(aabb=aabb)
```
I used the poster_v3 dataset to test the issue.
- The result of the default setting (scale is auto, aabb is -1~1) for splatfacto is as follows,
![image](https://github.com/user-attachments/assets/39788795-87d8-45b6-bbdc-98785de26b13)
- However, the result of the non-normalization setting (scale is 1, aabb is [[-44.0906, -44.3148, -42.1914],
        [ 44.0906,  44.3148,  42.1914]]) for splatfacto is as follows,
![image](https://github.com/user-attachments/assets/f2495aec-db29-4aa0-807b-fe122146190d)

As far as I know, the original Gaussian Splating is a non-normalization setting for poses and points and does not set the aabb scene box. Could you implement the issue and check the reason？
